easybuild/easyconfigs/m/mrc/mrc-1.3.13-GCCcore-13.3.0.eb

```diff
@@ -0,0 +1,50 @@
+# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2025/01
+easyblock = 'CMakeMake'
+
+name = 'mrc'
+version = '1.3.13'
+
+homepage = 'https://github.com/mhekkel/mrc'
+description = """Maartens Resource Compiler to store resources in the static section of an ELF binary."""
+
+Resources were mildly popular, other OS'es copied the concept in one way or     
+another. The company that invented them abandoned the whole concept though, they
+thought they had something better....
+
+Anyway, I still like being able to provide a single executable to the users of  
+my software. And given the usefulness of resources I decided to create a        
+compiler for them that works with the ELF executable format. Since using        
+resource forks is not an option I decided to store the data in the static data  
+section of an executable. The data is then available through global variables.
+"""
```

Comment:
Did you forget to remove the long description part?
This is now a syntax error

Reply:
I did